frappe/recorder.py

```diff
@@ -47,6 +47,8 @@ def get_current_stack_frames():
 		current = inspect.currentframe()
 		frames = inspect.getouterframes(current, context=10)
 		for frame, filename, lineno, function, context, index in list(reversed(frames))[:-2]:
+			if filename.endswith("recorder.py") and function == "post_process":
+				continue
```

Comment:
umm, better to handle this in test than to add a edge case handling here for it.